### ISSUE TITLE
-Wsometimes-uninitialized in drivers/gpu/drm/omapdrm/omap_fbdev.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1809]
```
drivers/gpu/drm/omapdrm/omap_fbdev.c:235:6: error: variable 'helper' is used uninitialized whenever 'if' condition is true [-Werror,-Wsometimes-uninitialized]
        if (!fbdev)
            ^~~~~~
drivers/gpu/drm/omapdrm/omap_fbdev.c:259:26: note: uninitialized use occurs here
        drm_fb_helper_unprepare(helper);
                                ^~~~~~
drivers/gpu/drm/omapdrm/omap_fbdev.c:235:2: note: remove the 'if' if its condition is always false
        if (!fbdev)
        ^~~~~~~~~~~
drivers/gpu/drm/omapdrm/omap_fbdev.c:228:30: note: initialize the variable 'helper' to silence this warning
        struct drm_fb_helper *helper;
                                    ^
                                     = NULL
1 error generated.
```

Patch submitted: https://lore.kernel.org/20230224-omapdrm-wsometimes-uninitialized-v1-1-3fec8906ee3a@kernel.org/

**Comment by nathanchance** [comment]
Patch accepted: https://cgit.freedesktop.org/drm/drm-misc/commit/?id=c176060a4c76ed0043cb9c10435af04ed1ad0560

**Comment by nathanchance** [comment]
This was merged into mainline before 6.3-rc1.
